source/sass/components/primary-nav.scss

```diff
@@ -250,7 +250,17 @@
           transition: transform 0.15s;
           display: inline-block;
           margin: 10px 0;
+          padding-bottom: 0;
+          border-width: 3px;
           color: $white;
+
+          @include hover-focus-active {
+            border-bottom-color: $white;
```

Comment:
Maybe it's been too long since I've written CSS, but how does an include statement also have a content block?

Reply:
oh this is a Bootstrap's mixin

```scss
@mixin hover-focus-active {
  @if $enable-hover-media-query {
    &:focus,
    &:active {
      @content
    }
    @include hover { @content }
  }
  @else {
    &:focus,
    &:active,
    &:hover {
      @content
    }
  }
}
```

Reply:
Ah, I forgot about the @content directive.